hy/core/shadow.hy

```diff
@@ -138,11 +138,17 @@
 (defn not-in [x y]
   (not-in x y))
 
+(defn get [thing key1 &rest keys]
+  (for* [k (+ (, key1) keys)]
+    (setv thing (get thing k)))
+  thing)
```

Comment:
So, I profiled my version against yours. (It helps that we fixed contrib/profile.) Mine is about twice as fast with one key, and still faster with two. Adding tuples is too slow for something this fundamental.